### PR TITLE
Example configuration.yaml entry

### PR DESCRIPTION
**Pull request by TheRedBull205**
# Proposed Changes
Adding a simple Example configuration.yaml entry to the existing config. For new users and people that are switching over from the official add-on.

### CODE REVIEW

**Comment by tjorim** [comment]
I disagree with the 'broker/hass ip adress', I think `a0d7b954-mqtt` should work just as well. (Or even `localhost` is better than an IP address, since the latter is different for each user.)

**Comment by TheRedBull205** [comment]
> 
> 
> I disagree with the 'broker/hass ip adress', I think `a0d7b954-mqtt` should work just as well. (Or even `localhost` is better than an IP address, since the latter is different for each user.)

I totaly agree, changed the PR.

**Comment by frenck** [comment]
Please use `a0d7b954-mqtt`, not `localhost`.

File: README.md [hunk 1]
```diff
@@ -67,7 +67,22 @@ first time starting the add-on).
...
 [![Docker Pulls][pulls-shield]][dockerhub]
 
-## Configuration
+## Configuration Home assistant
+
+**Note**: _Remember to restart the add-on when the configuration is changed._
+
+**Note2**: _When changing from the official add-on to this one, make sure to change configuration.yaml broker ip. "broker: core-mosquitto" to "broker: localhost"_
+
+```json
```
Comment (ludeeus) [inline README.md]:
```suggestion
```yaml
```

File: README.md [hunk 2]
```diff
@@ -67,7 +67,22 @@ first time starting the add-on).
 [![Docker Layers][layers-shield]][microbadger]
 [![Docker Pulls][pulls-shield]][dockerhub]
 
-## Configuration
+## Configuration Home assistant
+
+**Note**: _Remember to restart the add-on when the configuration is changed._
```
Comment (ludeeus) [inline README.md]:
```suggestion
### Notes
_Remember to restart the add-on when the configuration is changed._

_If you are moving from the official add-on to this one, make sure that you change the `broker:` in your configuration from `core-mosquitto` to `a0d7b954-mqtt`._

```

File: README.md [hunk 3]
```diff
@@ -67,7 +67,22 @@ first time starting the add-on).
...
+
+**Note2**: _When changing from the official add-on to this one, make sure to change configuration.yaml broker ip. "broker: core-mosquitto" to "broker: localhost"_
+
+```json
+# Example configuration.yaml entry
+mqtt:
+  broker: localhost
```
Comment (ludeeus) [inline README.md]:
```suggestion
  broker: a0d7b954-mqtt
```

File: README.md [hunk 4]
```diff
@@ -67,7 +67,22 @@ first time starting the add-on).
...
+  username: !secret mqtt_username
+  password: !secret mqtt_password
+  client_id: home-assistant
+```
+
+## Configuration add-on
```
Comment (ludeeus) [inline README.md]:
```suggestion
## Add-on configuration example
```

File: README.md [hunk 5]
```diff
@@ -67,7 +67,22 @@ first time starting the add-on).
 [![Docker Layers][layers-shield]][microbadger]
 [![Docker Pulls][pulls-shield]][dockerhub]
 
-## Configuration
+## Configuration Home assistant
```
Comment (ludeeus) [inline README.md]:
```suggestion
## Home Assistant configuration example
```

File: README.md [hunk 6]
```diff
@@ -67,7 +67,22 @@ first time starting the add-on).
...
+
+**Note**: _Remember to restart the add-on when the configuration is changed._
+
+**Note2**: _When changing from the official add-on to this one, make sure to change configuration.yaml broker ip. "broker: core-mosquitto" to "broker: localhost"_
```
Comment (ludeeus) [inline README.md]:
```suggestion
```

File: README.md (hunk 3 above)
Comment (PixelJonas) [reply README.md]: as a newbie who is trying to connect this addon with HA. What's the meaning of `a0d7b954`  ?

Comment (tjorim) [reply README.md]: It's just part of the name to address the add-on: https://developers.home-assistant.io/docs/en/hassio_addon_communication.html#network

File: README.md (hunk 6 above)
Comment (TheRedBull205) [reply README.md]: Didnt remove the entire note, just the last part since that should be obvious.

**Comment by TheRedBull205** [comment]
@ludeeus I hope this is better 👍

**Comment by ludeeus** [comment]
Looks good now :)
Thanks 🎉

**Pull request merged**